service/controller/resource/alerting/alertmanager/resource.go

```diff
@@ -173,6 +173,9 @@ func toAlertmanager(v interface{}, config Config) (metav1.Object, error) {
 					},
 				},
 			},
+			AlertmanagerConfigNamespaceSelector: &metav1.LabelSelector{
+				MatchLabels: key.NamespaceLabels(),
```

Comment:
We match the alertmanagerconfigs in namespaces managed by prometheus-meta-operator